db/seeds.d/020-provisioning_templates_list.rb

```diff
@@ -94,13 +109,17 @@ def seeded_templates
     end
 
     def kinds
-      kinds = {}
-      TemplateKind.default_template_labels.keys.map(&:to_sym).each do |type|
-        kinds[type] = TemplateKind.unscoped.find_by_name(type)
-        kinds[type] ||= TemplateKind.unscoped.create(:name => type)
-        raise "Unable to create template kind: #{format_errors kinds[type]}" if kinds[type].nil? || kinds[type].errors.any?
+      @kinds ||= begin
+        @kinds ||= {}
+        TemplateKind.default_template_labels.keys.map(&:to_sym).each do |type|
+          return @kinds[type] if @kinds.key?(type)
+          @kinds[type] ||= TemplateKind.unscoped.find_or_create_by(name: type)
+          @kinds[type].description = template_description[@kinds[type].name]
+          @kinds[type].save
+          raise "Unable to create template kind: #{format_errors @kinds[type]}" if @kinds[type].errors.any?
+        end
+        @kinds
       end
-      kinds
     end
```

Comment:
This is what I meant. But do not access `@kind` inside of the block, just use local variable and return it.